### ISSUE TITLE
home page fail to load

### ISSUE THREAD

**Issue by MrYogi** [balzathor/UltimateBrowserProject#87]
when app is on Home page, remove app from recent apps list. Open again app, then blank page will load, even address bar doesn't go to any link typed.

**Comment by xdevs23** [comment]
Ok thanks for your report, we may analize this when we work on UBP again later maybe.

**Comment by Thunderbottom** [comment]
Works fine, what version of app are you on?

**Comment by MrYogi** [comment]
v2.0.1, address bar starts working if multi tabs are not collapsed

**Comment by Thunderbottom** [comment]
Yes, every version after 1.4.6 is very buggy and was meant to be experimental. I have plans to rewrite the code once I'm done with exams so... this will be fixed sometime this year :)

**Comment by MrYogi** [comment]
as you suggested i rolled back to v1.4.6 and it really seems more stable, and all functionalities working fine. Just one issue i am facing of downloading files on marshmallow devices. I know that you removed that bug in v2(unable to find what changes you done), can you give me hint of what changes required for that so that i can make those changes in v1.4.6 to achieve most stable version.
thanks

**Comment by xdevs23** [comment]
You need to grant storage permission for UBP.

**Comment by MrYogi** [comment]
thanks for quick reply

**Comment by xdevs23** [comment]
Np, please report if it worked.

**Comment by MrYogi** [comment]
yes, it worked

**Comment by xdevs23** [comment]
Great, thx.

**Comment by Thunderbottom** [comment]
lol, We need to add permission settings I guess!!

**Comment by xdevs23** [comment]
We have added already, it wasn't added in 1.4.6

**Comment by Thunderbottom** [comment]
no, in 1.4.6 I mean lol
